tests/scripts/components-compliance.sh

```diff
@@ -0,0 +1,28 @@
+# components-compliance.sh
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+# This file contains test components that are executed by all.sh
+
+################################################################
+#### Compliance Testing
+################################################################
+
+component_test_psa_compliance () {
```

Comment:
```suggestion
component_tf_psa_crypto_test_psa_compliance () {
```
mbedtls and TF-PSA-Crypto all.sh components need to have different names for the time being.

Reply:
Ops, sorry, I wasn't aware of this and I forgot to check other components. It should be fixed now.